apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/BlockTilSlateKonverterer.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.etterlatte.brev
+
+import no.nav.etterlatte.brev.model.Slate
+import no.nav.pensjon.brevbaker.api.model.RenderedJsonLetter
+
+object BlockTilSlateKonverterer {
+    fun konverter(it: RenderedJsonLetter) = Slate(
+        it
+            .blocks
+            .map { block -> tilSlateElement(block) }
+            .toList()
+    )
+
+    private fun tilSlateElement(block: RenderedJsonLetter.Block) = when (block.type) {
+        RenderedJsonLetter.Block.Type.TITLE1 -> TODO()
```

Comment:
Er det tenkt at denne foreløpig ikke skal støttes? Hadde vært fin med en `TODO("Stopper konvertering til slate siden vi fikk Title1 i innhold, som ikke skal skje")` f.eks., med grunnen for at den ikke skal være med :)

Reply:
Ja, eg trur ikkje det skal skje/skal kunne skje, og ser ikkje heilt kva vi i så fall skal mappe det til. Liker konseptet med betre feilmelding da, tar inn den.

Reply:
Eller vent, i brevbaker-koden konverterer vi heading_two til title1 og heading_three til title2... Oppdaterer